api.planx.uk/modules/send/utils/exportZip.ts

```diff
@@ -206,6 +207,11 @@ export class ExportZip {
     this.zip = new AdmZip();
     // make a tmp directory to avoid file name collisions if simultaneous applications
     this.tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), sessionId));
+
+    // we can't directly access `__dirname` in ESM, so get equivalent using fileURLToPath
+    const __filename = fileURLToPath(import.meta.url); // get the resolved path to the file
+    const __dirname = path.dirname(__filename); // get the name of the directory
```

Comment:
This change is related to recent `esnext` migration & failing e2e tests. 

It works exactly as expected but throws testing error `The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'es2022', 'esnext', 'system', 'node16', or 'nodenext'.` which I can't quite make sense of because our API's tsconfig.json `module` does indeed match these supported options